opm/simulators/flow/FlowGenericVanguard.hpp

```diff
@@ -284,6 +284,8 @@ class FlowGenericVanguard {
     void updateOutputDir_(std::string outputDir,
                           bool enableEclCompatFile);
 
+    void updateNOSIM_(std::string enableDryRun);
```

Comment:
can be const std::string&

Reply:
I have rebased with std::string_view, thanks. Is there something else to be changed before calling Jenkins?